packages/core/test/SignUp.ts

```diff
@@ -80,5 +84,38 @@ describe('Signup', function () {
                 'ReachedMaximumNumberUserSignedUp'
             )
         })
+
+        it('setUsername should succeed', async () => {
+            const tx = await unirepSocialContract.setUsername(
+                config.NUM_EPOCH_KEY_NONCE_PER_EPOCH,
```

Comment:
The first argument to this function is the epoch key that is being attested to. You should use a random value for this, or even just a hardcoded value like `0x2140`. There's a function in `@unirep/crypto` called `genRandomSalt()`. You should able to use that to get safe random epoch keys.

Definitely don't this config value here, it doesn't really make sense.

Reply:
Is it ok to set epochKey's type as `epochKey: BigNumberish | BigInt` ? cuz currently `BigNumberish` does not include bigint/BigInt. ref: https://github.com/ethers-io/ethers.js/discussions/1471 
You get this error saying `Argument of type 'BigInt' is not assignable to parameter of type 'BigNumberish'.
  Type 'BigInt' is missing the following properties from type 'BigNumber': _hex, _isBigNumber, fromTwos, toTwos, and 24 more.`  with `genRandomSalt()` which returns BigInt